pkg/exporter/exporter.go

```diff
@@ -164,6 +180,12 @@ func New(server string, timeout time.Duration, logger log.Logger) *Exporter {
 			nil,
 			nil,
 		),
+		rejectedConnections: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, "", "rejected_connections_total"),
+			"Conns rejected in maxconns_fast mode.",
```

Comment:
As a bonus and for consistency it would be great to align the description style with existing metric descriptions. See the description of connections_yielded_total for an example.

Reply:
Thank you for reviewing!